src/hal/src/command/raw.rs

```diff
@@ -16,7 +16,7 @@ use super::{
 ///
 pub trait RawCommandBuffer<B: Backend>: Clone + Send {
     ///
-    fn begin(&mut self);
+    fn begin(&mut self, reusable: bool);
```

Comment:
Could we use flags here as we will also want to support simultaneous usage and renderpass only command buffers.

Reply:
Yeah, in my local repository I turned this into flags once I started working on secondary buffers.